integration_tests/tests/forseti/controls/scanner-config_validator_gcs_policy.rb

```diff
@@ -54,20 +54,33 @@
   # Act
   describe command("forseti scanner run") do
     its('exit_status') { should eq 0 }
-    its('stdout') { should match (/Running ConfigValidatorScanner.../) }
-    its('stdout') { should match (/Scan completed/) }
-    its('stdout') { should match (/Scanner Index ID: (.*[0-9]*) is created/) }
+    its('stdout') { should match(/Running ConfigValidatorScanner.../) }
+    its('stdout') { should match(/Scan completed/) }
+    its('stdout') { should match(/Scanner Index ID: (.*[0-9]*) is created/) }
+  end
+
+  # Test notifier for the GCS export
+  describe command("forseti notifier run --inventory_index_id #{@inventory_id}") do
```

Comment:
Adding this command so that we can assert the violations are picked up by Notifier.